### PR TITLE
sorting accented characters

### PR DESCRIPTION
**Pull request by maryamsulemani97**
Mention that strings are sorted lexicographically based on their byte value

### CODE REVIEW

File: learn/advanced/sorting.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ To allow your users to sort results at search time you must:
 2. Add those attributes to the `sortableAttributes` index setting
 3. Update Meilisearch's [ranking rules](/learn/core_concepts/relevancy.md) (optional)
 
+::: note
+Meilisearch sorts strings in lexicographic order based on their byte values. This means, `á` which has a value of 225, will be sorted after `z`, which has a value of 122.
```
Comment (dichotommy) [inline learn/advanced/sorting.md:20]:
```suggestion
Meilisearch sorts strings in lexicographic order based on their byte values. For example, `á`, which has a value of 225, will be sorted after `z`, which has a value of 122.
```

**Review by dichotommy — COMMENTED** [review]
Could you link to an issue or similar info source?

I find it hard to believe that this is truly the behavior of string sorting, since it means that "this" and "This" would not be sorted together; based on byte values, uppercase letters would all be sorted first followed by lowercase letters. Am I misunderstanding something? 🤔

**Comment by maryamsulemani97** [comment]
>Could you link to an issue or similar info source?

Issue: https://github.com/meilisearch/meilisearch/issues/3260
Slack thread: https://meilisearch.slack.com/archives/C010VV3SHQF/p1671724281502099?thread_ts=1671718450.818539&cid=C010VV3SHQF

File: learn/advanced/sorting.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ To allow your users to sort results at search time you must:
 3. Update Meilisearch's [ranking rules](/learn/core_concepts/relevancy.md) (optional)
 
 ::: note
-Meilisearch sorts strings in lexicographic order based on their byte values. For example, `á`, which has a value of 225, will be sorted after `z`, which has a value of 122.
+Documents used for sort are converted to lowercase and then sorted in lexicographic order based on their byte values. For example, `á`, which has a value of 225, will be sorted after `z`, which has a value of 122.
```
Comment (dichotommy) [inline learn/advanced/sorting.md:20]: For me this wording raises the question of whether the fields in the returned documents will actually be lowercased. I'm guessing that they're only lowercased for the purposes of sorting and will be unchanged in the results. If so, could you rephrase to make that clear?

File: learn/advanced/sorting.md [hunk 3]
```diff
@@ -16,6 +16,10 @@ To allow your users to sort results at search time you must:
 2. Add those attributes to the `sortableAttributes` index setting
 3. Update Meilisearch's [ranking rules](/learn/core_concepts/relevancy.md) (optional)
 
+::: note
+Meilisearch converts documents to lowercase for sorting, they remain unchanged in the results. They are then sorted in lexicographic order based on their byte values. For example, `á`, which has a value of 225, will be sorted after `z`, which has a value of 122.
```
[the remark this answers is not in the document]
Comment (maryamsulemani97) [reply learn/advanced/sorting.md]: That's a lot clearer. Thanks

**Review by dichotommy — APPROVED** [review]
🧑‍🚀 👍 🔥 🚀

**Comment by maryamsulemani97** [comment]
bors merge
